src/runtime/cl_functions.h

```diff
@@ -213,140 +128,36 @@ CL_FN(cl_int,
-                           size_t           /* offset */,
-                           size_t           /* size */,
-                           cl_uint          /* num_events_in_wait_list */,
-                           const cl_event * /* event_wait_list */,
-                           cl_event *       /* event */,
-                           cl_int *         /* errcode_ret */));
+      clEnqueueMapBuffer, (cl_command_queue /* command_queue */, cl_mem /* buffer */, cl_bool /* blocking_map */, cl_map_flags /* map_flags */, size_t /* offset */, size_t /* size */, cl_uint /* num_events_in_wait_list */, const cl_event * /* event_wait_list */, cl_event * /* event */, cl_int * /* errcode_ret */));
 
 CL_FN(void *,
-      clEnqueueMapImage, (cl_command_queue  /* command_queue */,
-                          cl_mem            /* image */,
-                          cl_bool           /* blocking_map */,
-                          cl_map_flags      /* map_flags */,
-                          const size_t *    /* origin[3] */,
-                          const size_t *    /* region[3] */,
-                          size_t *          /* image_row_pitch */,
-                          size_t *          /* image_slice_pitch */,
-                          cl_uint           /* num_events_in_wait_list */,
-                          const cl_event *  /* event_wait_list */,
-                          cl_event *        /* event */,
-                          cl_int *          /* errcode_ret */));
-
-CL_FN(cl_int,
-      clEnqueueUnmapMemObject, (cl_command_queue /* command_queue */,
-                                cl_mem           /* memobj */,
-                                void *           /* mapped_ptr */,
-                                cl_uint          /* num_events_in_wait_list */,
-                                const cl_event *  /* event_wait_list */,
-                                cl_event *        /* event */));
-
-CL_FN(cl_int,
-      clEnqueueNDRangeKernel, (cl_command_queue /* command_queue */,
-                               cl_kernel        /* kernel */,
-                               cl_uint          /* work_dim */,
-                               const size_t *   /* global_work_offset */,
-                               const size_t *   /* global_work_size */,
-                               const size_t *   /* local_work_size */,
-                               cl_uint          /* num_events_in_wait_list */,
-                               const cl_event * /* event_wait_list */,
-                               cl_event *       /* event */));
+      clEnqueueMapImage, (cl_command_queue /* command_queue */, cl_mem /* image */, cl_bool /* blocking_map */, cl_map_flags /* map_flags */, const size_t * /* origin[3] */, const size_t * /* region[3] */, size_t * /* image_row_pitch */, size_t * /* image_slice_pitch */, cl_uint /* num_events_in_wait_list */, const cl_event * /* event_wait_list */, cl_event * /* event */, cl_int * /* errcode_ret */));
```

Comment:
The changes to this file seem like a pessimizaton

Reply:
Reverted whole file, marked with 'clang-format off'